apps/src/templates/sectionProgressV2/IconKey.jsx

```diff
@@ -7,23 +7,30 @@ import AssignmentCompletionStatesBox from './AssignmentCompletionStatesBox';
 import styles from './progress-table-legend.module.scss';
 import {Heading6} from '@cdo/apps/componentLibrary/typography';
 import FontAwesome from '../FontAwesome';
+import Link from '@cdo/apps/componentLibrary/link';
+import MoreDetailsDialog from './MoreDetailsDialog.jsx';
 
 export default function IconKey({isViewingValidatedLevel, expandedLessonIds}) {
   const [isOpen, setIsOpen] = useState(false);
+  const [isIconDetailsOpen, setIconDetailsOpen] = useState(false);
+
+  const toggleIsViewingDetails = event => {
+    event.preventDefault();
```

Comment:
The `Link` component re-renders the page (or parts of the page) which we don't want.  This prevents that from happening.